index.html

```diff
@@ -1,51 +1,306 @@
-              </button>
-            </nav>
+        <div class="logo">
+          <a href="#" id="name">Nimrod Acosta</a>
+        </div>
+        <nav class="burger">
+          <button class="normalb">
+            <img src="./Images/Union.png" class="union" alt="" />
+          </button>
+        </nav>
       </div>
-      
-
-  </header>
-  <section class="continer">
+    </header>
+    <!--Section Home-->
+    <section class="continer">
       <h1 class="title">Hey There. I’m Nimrod</h1>
       <h2 class="subtitle">I’am a Software Developer</h2>
-      <p class="paragraph">Encoding for life. <br> 
-        Web Solutions and Mobile Applications</p>
+      <p class="paragraph">
+        Encoding for life. <br />
+        Web Solutions and Mobile Applications
+      </p>
       <ul class="social">
-          <li class="socialimg">
-            <img src="./Images/Github.png" alt="">
-          </li>
-          <li class="socialimg">
-            <img src="./Images/Icon-Linkedin.png" alt="">
-          </li>
-          <li class="socialimg">          
-            <img src="./Images/hand.png" alt="">
-          </li>
-          <li class="socialimg">
-            <img src="./Images/Icon-Twitter.png" alt="">
-          </li>
-          <li class="socialimg">
-            <img src="./Images/Icon-Medium.png" alt="">
-          </li>
-        </ul>                      
-  </section>    
-</body>
-</html>
\ No newline at end of file
+        <li class="socialimg">
+          <img src="./Images/Github.png" alt="" />
+        </li>
+        <li class="socialimg">
+          <img src="./Images/Icon-Linkedin.png" alt="" />
+        </li>
+        <li class="socialimg">
+          <img src="./Images/hand.png" alt="" />
+        </li>
+        <li class="socialimg">
+          <img src="./Images/Icon-Twitter.png" alt="" />
+        </li>
+        <li class="socialimg">
+          <img src="./Images/Icon-Medium.png" alt="" />
+        </li>
+      </ul>
+    </section>
+    <!--Section Works-->
+    <section class="works">
+      <div id="wheader">
+        <h1 id="wtitle">My Recent Works</h1>
+        <div id="line"></div>
+      </div>
+      <div class="workintroduction">
+        <div id="wintro">
+          <img src="./Images/Placeholder.png" alt="" id="wimage" />
+          <div id="wpost">
+            <h2 id="wh2">Multi-Post Stories</h2>
+            <p id="wpar">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul id="wlist">
+              <li class="wiconlist">css</li>
+              <li class="wiconlist">html</li>
+              <li class="wiconlist">bootstrap</li>
+              <li class="wiconlist">Ruby</li>
+            </ul>
```

Comment:
- Please follow the Figma design details, I noticed in your technologies used for the project, you have text color as black, and the background color of the container does not correspond to the Figma design.
**Note:**
- text color is `#FFFFFF`
- Conatiner color is `background: rgba(255, 255, 255, 0.24);`
- font-family `font-family: 'Inter';`
- font size is `15px`

##### Figma design
![Screenshot from 2022-05-28 08-47-42](https://user-images.githubusercontent.com/58771507/170816107-68c555b8-ec73-4000-80a6-92d1b0ef2743.png)

##### Your design
![Screenshot from 2022-05-28 08-48-20](https://user-images.githubusercontent.com/58771507/170816133-8a7354c4-f89b-446e-b002-16a610cf5238.png)